static/app/views/dashboards/widgetBuilder/buildSteps/filterResultsStep/releaseSearchBar.tsx

```diff
@@ -75,7 +81,26 @@ export function ReleaseSearchBar({
     return acc;
   }, {});
 
-  return (
+  return organization.features.includes('search-query-builder-releases') ? (
+    <SearchQueryBuilder
+      initialQuery={widgetQuery.conditions}
+      filterKeySections={filterKeySections}
+      filterKeys={supportedTags}
+      getTagValues={memoize(
+        getTagValues,
+        ({key}, searchQuery) => `${key}-${searchQuery}`
+      )}
+      placeholder={t('Search for release version, session status, and more')}
+      onChange={(query, state) => {
+        onClose?.(query, {validSearch: state.queryIsValid});
+      }}
+      searchSource="widget_builder"
+      disallowWildcard
+      disallowUnsupportedFilters
+      disallowFreeText
+      recentSearches={SavedSearchType.SESSION}
+    />
```

Comment:
Do you need to pass the `invalidMessages` here? The previous version had that and we still support that prop

Reply:
I thought the default messages were pretty good, but I added explicit ones 👍🏻